src/main/java/com/o19s/es/explore/ExplorerScorer.java

```diff
@@ -81,7 +81,11 @@ public int docID() {
 
     @Override
     public DocIdSetIterator iterator() {
-        return subScorer.iterator();
+        try {
+            return subScorer.iterator();
+        } catch (Exception e) {
```

Comment:
we probably don't need this
I suppose you wanted to catch a NullPointerException but I suspect that you already fixed this NPE by returning null directly from ExplorerQuery.ExplorerWeight::scorer instead of creating this scorer

If you added this because of something else we should fix the root cause instead.